src/favoriteManager.js

```diff
@@ -1,16 +1,17 @@
-export function toggle(mentor) {
-  const favs = get();
+import { getFavorites, addMentorToFavorites } from './api';
+
+export function toggle(mentor, favs) {
   const favIndex = favs.indexOf(mentor._id);
 
   if (favIndex > -1) {
     favs.splice(favIndex, 1);
   } else {
     favs.push(mentor._id);
   }
-  localStorage.setItem('favs', JSON.stringify(favs));
+  addMentorToFavorites(mentor._id);
   return favs;
 }
 
-export function get() {
-  return JSON.parse(localStorage.getItem('favs') || '[]');
+export async function get() {
+  return await getFavorites();
 }
```

Comment:
Don't need to use `async`/`await`. If getFavorites is already a `Promise`, is what it will be returned. So, just execute:
```
export function get() {
  return getFavorites()
}
```

Reply:
On the other hand, the execution of `get()` has been removed. So, this function declaration could be removed as well.

Reply:
> On the other hand, the execution of `get()` has been removed. So, this function declaration could be removed as well.

I'm not sure I understood the comment but I meant that `favoriteManager` will be response of all the favorites functionality for separation of concerns, even the `get` function just return the result from the API.